crates/util/src/task.rs

```diff
@@ -0,0 +1,129 @@
+use std::future::Future;
+use std::sync::LazyLock;
+
+use tokio_util::sync::CancellationToken;
+use tokio_util::task::TaskTracker;
+
+pub trait FutureOutputExt {
```

Comment:
It's pretty important and is used here:

https://github.com/eqlabs/pathfinder/pull/2456/files#diff-b28a8f63ef5d07b8eace6f183eb569a37beee865a81944de2857f49e9a9353a3R38

The aim is to avoid another `?` when joining gracefully cancelled tasks and utilize the most common return types from such tasks, which are either `()`, or `anyhow::Result`.